packages/now-routing-utils/src/superstatic.ts

```diff
@@ -218,18 +218,42 @@ function replaceSegments(
   return destination;
 }
 
-function safelyCompile(str: string, indexes: { [k: string]: string }): string {
-  if (!str) {
-    return str;
+function safelyCompile(
+  value: string,
+  indexes: { [k: string]: string }
+): string {
+  if (!value) {
+    return value;
   }
-  // path-to-regexp cannot compile question marks
-  return str
-    .split('?')
-    .map(part => {
-      const compiler = compile(part);
-      return compiler(indexes);
-    })
-    .join('?');
+
+  for (const key of Object.keys(indexes)) {
+    if (value.includes(`:${key}`)) {
+      value = value
+        .replace(
+          new RegExp(`:${key}\\*`, 'g'),
+          `:${key}--ESCAPED_PARAM_ASTERISKS`
```

Comment:
Nit pick: this is plural while the others are singular. Perhaps remove the `S`?

Reply:
Updated